api/v1/placementbinding_types.go

```diff
@@ -64,21 +81,21 @@ type PlacementBinding struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 	// +kubebuilder:validation:Optional
 	BindingOverrides BindingOverrides `json:"bindingOverrides,omitempty"`
-	// This field provides the ability to select a subset of bound clusters
+	// SubFilter provides the ability to select a subset of bound clusters. This parameter is
+	// optional. If you set it, you must set it to "restricted".
```

Comment:
Could you define what "restricted" does?

Reply:
No? 😆 I'll look back over the enhancement and add some details.

Reply:
Okay, I think I've captured it. I couldn't tell you how many times I've tried to work this one out prior to this--it seems to me it's configuring overrides for overrides, which is hella confusing. 🤪 Let me know whether it makes sense to you.